templates/solutions/telco/5g-core.html

```diff
@@ -0,0 +1,589 @@
+                  class="u-embedded-media__element"
+                  src="https://www.youtube.com/embed/tNECRG5r2sg?si=B98-42RGdgpJebjQ"
+                  allowfullscreen=""></iframe>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <div class="row">
+      <hr />
+      <div class="col-3">
+        <div class="p-section--shallow">
+          <h2>Learn more</h2>
+        </div>
+      </div>
+      <div class="col-9">
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Webinars</h3>
+            </div>
+            <div class="col-6">
+              <ul class="p-list--divided">
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/engage/telco-cloud-implementation">Telco cloud implementation</a>
+                </li>
+                <li class="p-list__item">
+                  <a href="https://bit.ly/3FW2QR2">Kubernetes on bare metal: ready for prime time!</a>
+                </li>
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/engage/improving-telecom-security-2023-webinar">Improving telecom security</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <hr class="p-rule--muted" />
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">Suggested reading</h3>
+            </div>
+            <div class="col-6">
+              <ul class="p-list--divided">
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/blog/telecom-security-how-to-safeguard-your-open-source-telco-infrastructure">Safeguarding your telco infrastructure with Ubuntu Pro</a>
+                </li>
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/engage/bare-metal-kubernetes">How to run workloads on bare metal Kubernetes with MAAS</a>
+                </li>
+                <li class="p-list__item">
+                  <a href="https://ubuntu.com/engage/telco-dilemma-reduce-5g-infra-cost">Overcome the telco dilemma: reduce 5G infrastructure costs with open source</a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="p-section">
+    <hr class="u-fixed-width" />
+    <div class="p-strip is-deep">
+      <div class="row u-vertically-center">
+        <div class="col">
+          <div class="p-section--shallow">
+            <h2>
+              Canonical provides a full stack for your telecom infrastructure.
+              <br />
+              To learn more about our telco solutions, get in touch.
+            </h2>
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <p>
+            <a href="https://ubuntu.com/telco#get-in-touch"
```

Comment:
Might be worth confirming with @Sophie-32 that this is the form the cta is supposed to route to. It's what's linked on the doc but it seems strange that we built a new modal for this page but we're sending people to u.com on this cta while on the other we're using the local modal

Reply:
I'll add the `js-invoke-modal` class to the CTA, will wait for @Sophie-32's input about the fallback route.

Reply:
@mtruj013 is correct, it should route to the same form used in the 'Contact us' CTA in the hero (its noted in the comments on the copydoc.